app/src/main/res/values/strings.xml

```diff
@@ -5,96 +5,183 @@
 
+    <!-- This is used as the label on a button on a first-run welcome screen to sign in and start using the application -->
     <string name="welcome_start_btn">Get Started</string>
-    <string name="welcome_instructions">To use Firefox Lockbox, you\'ll need a\nFirefox Account with saved logins.</string>
-
+    <!-- This provides context near a button on the first-run welcome screen to describe application requirements, not expected to translate Firefox Account nor Firefox Lockbox -->
+    <string name="welcome_instructions">To use Firefox Lockbox, you’ll need a\nFirefox Account with saved logins.</string>
+    <!-- This is the link on the first-run welcome screen to learn more about the requirements of the application -->
     <string name="welcome_learn_more_btn">Learn More</string>
+    
+    <!-- This is the label for a button only available on test builds to skip the authentication step and immediately use test data -->
+    <string name="placeholder_skip_fxa">Use test data</string>
+
+    <!-- This is the label used by screenreaders to inform the user the navigation button selected is to go backwards from where they came from -->
+    <string name="backable_description">Back</string>
+    <!-- This is the label used by screenreaders to inform the user the button selected is meant to open a navigation menu --> 
+    <string name="menu_description">Menu</string>
+
+    <!-- This is the label for the navigation item that opens a Frequently Asked Questions web page -->
     <string name="nav_menu_faq">FAQ</string>
+    <!-- This is the label for the navigation item that opens a survey or feedback web page to submit to the developers -->
     <string name="nav_menu_feedback">Provide Feedback</string>
+    <!-- This is the label for the navigation item that opens a screen to manage the user account currently signed in to the application -->
     <string name="nav_menu_account">Account</string>
+    <!-- This is the label for the navigation item that opens a settings screen with options and preferences and more application information -->
     <string name="nav_menu_settings">Settings</string>
+    <!-- This is the label for a button, primarily in the navigation drawer, that securely locks the application requiring the user to unlock it -->
+    <string name="lock_now">Lock Now</string>
 
-    <string name="sort_menu_az">Alphabetically</string>
-    <string name="sort_menu_recent">Recently Used</string>
-
+    <!-- This label is used by screenreaders to inform the user they have selected an image of a lock -->
+    <string name="lock_icon_description">Lock icon image</string>
+    <!-- This is the label for the button to unlock the application for use while on a lock screen -->
     <string name="unlock_button">Unlock</string>
-    <string name="lock_now">Lock Now</string>
+
+    <!-- This message appears in a toast informing the user it is currently downloading and syncing entry data -->
```

Comment:
>It may be worth defining very simply/briefly what "entries" here refers to exactly, since localizers will not all familiar with this project

@Delphine OK! I added a quick message to the first occurrence. Is it necessary to repeat it throughout?